packages/sample-app/sample_app/strands_example.py

```
@@ -0,0 +1,410 @@
+                     "- Real weather forecasts and climate data\n" +
+                     "- Up-to-date travel advisories and requirements\n" +
+                     "- Current events and local information\n" +
+                     "Always provide accurate, real-time travel information."
+    )
+    
+    # Planning Agent - Specialized in itinerary creation
+    planning_agent = Agent(
+        model=model,
+        tools=[create_itinerary, estimate_budget, get_memory_context, save_trip_plan],
+        system_prompt="You are a Trip Planning Specialist. Your expertise includes:\n" +
+                     "- Creating detailed, personalized itineraries\n" +
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)